testsuite/mpitests/test_mini_brunel_ps.sli

```diff
@@ -36,6 +36,7 @@ See: brunel_ps.sli
 
 (unittest) run
 /unittest using
+/distributed_process_invariant_events_assert_or_die << /show_results true >> SetOptions
```

Comment:
With debugging complete, I would turn this false again.
```suggestion
/distributed_process_invariant_events_assert_or_die << /show_results false >> SetOptions
```

Reply:
Keeping this true for now due to the MPI heisenbug.